src/popover.tsx

```diff
@@ -278,14 +257,24 @@ const Popover: React.FC<Props> = ({
                 <div className={classes.arrowWrapper} style={arrowStyles}>
                     <div className={classes.arrow} />
                 </div>
-                <div className={classes.boxContainer}>
-                    {asset && <div className={classes.assetContent}>{asset}</div>}
-                    <div className={classes.textContent}>
-                        {title && <span className={classes.title}>{title}</span>}
-                        <span className={classes.text}>{description}</span>
-                    </div>
-                    <div className={classes.closeButtonIcon}>
+                <Box padding={16}>
+                    <Stack space={0}>
+                        <Box paddingRight={24}>
+                            <Inline space={16}>
+                                {asset}
+                                <Box className={classes.textAlign}>
```

Comment:
This box is for align items to the center in this case
https://ibit.ly/zBne

I couldn't solve without this, but maybe there is exist another workaround